tsp-output/@azure-tools/typespec-csharp/src/Generated/Internal/ModelSerializationExtensions.cs

```diff
@@ -257,7 +257,7 @@ internal static class TypeFormatters
             public static string ToString(DateTime value, string format) => value.Kind switch
             {
                 DateTimeKind.Utc => ToString((DateTimeOffset)value, format),
-                _ => throw new NotSupportedException($"DateTime {value} has a Kind of {value.Kind}. Azure SDK requires it to be UTC. You can call DateTime.SpecifyKind to change Kind property value to DateTimeKind.Utc.")
+                _ => throw new NotSupportedException($"DateTime {value} has a Kind of {value.Kind}. Generated clients require it to be UTC. You can call DateTime.SpecifyKind to change Kind property value to DateTimeKind.Utc.")
```

Comment:
This change is resolved in this PR: https://github.com/Azure/autorest.csharp/issues/4485